src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.net46.cs

```diff
@@ -8,17 +8,17 @@ namespace System.IO
 {
     public static class FileSystemAclExtensions
     {
-        public static DirectorySecurity GetAccessControl(DirectoryInfo directoryInfo)
+        public static DirectorySecurity GetAccessControl(this DirectoryInfo directoryInfo)
```

Comment:
I thought about it and I think ReferenceException is indeed the correction exception if this is null.